packages/@coorpacademy-app-review/src/reducers/data/slides.ts

```diff
@@ -1,50 +1,23 @@
-import includes from 'lodash/fp/includes';
+import has from 'lodash/fp/has';
+import set from 'lodash/fp/set';
 import {SlidesAction} from '../../actions/data/slides';
 import {SLIDE_FETCH_SUCCESS} from '../../actions/api/fetch-slide';
 import {SlideFromAPI} from '../../types/common';
 
-export type SlidesState = {
-  slideRefs: string[];
-  values: Record<string, SlideFromAPI>;
-};
-
-const getInitialState = (): SlidesState => {
-  const state: SlidesState = {
-    slideRefs: [],
-    values: {}
-  };
-  return state;
-};
-
-export const initialState: SlidesState = getInitialState();
-
-// -----------------------------------------------------------------------------
-
-const setSlide = (state: SlidesState, slide: SlideFromAPI): SlidesState => {
-  const slideRefs = state.slideRefs;
-  if (!includes(slide.universalRef, slideRefs)) slideRefs.push(slide.universalRef);
-  const _state: SlidesState = {
-    slideRefs,
-    values: {
-      ...state.values,
-      [slide.universalRef]: slide
-    }
-  };
-
-  return _state;
-};
+export type SlidesState = Record<string, SlideFromAPI>;
```

Comment:
```suggestion
export type SlidesState = {
  slideRefs: string[];
  values: Record<string, SlideFromAPI>;
};
```
on a besoin de la prop slideRefs, c'est un tableau de string refs (vide au début) qui garde l'ordre de fetch des slides (c'est comme ça que on fonction de la current ref on peux calculer la position de la slide) j'ai besoin de ça car un objet ne garanti pas un ordre de insertion des clés et je ne peux pas savoir la position sans ça

Reply:
L'ordre des slides est disponible dans la progression `state.slides`